### ISSUE TITLE
Improve the import failure error description

### ISSUE THREAD

**Issue by hmartiins** [nullstack/nullstack#81]
**Suggested improvement:**
A better description of error when not finding imported file.

**Describing my experience to reach such a conclusion:**
When I was working on my Nullstack project, I created a component and then changed the file and folder name of that component, but I forgot to modify the import for that component, thus generating an error in the application because it was not possible to import the component, and only thing printed on the terminal was:
```
💥️ There is an error preventing compilation in ./src/Application.njs
    @ ./src/Application.njs 3:0-32 24:92-99
    @ ./index.js
```
So I spent some time trying to figure out what this error was, it would be much more productive if the error thrown in the terminal pointed out that it was not possible to compile the program because it was not possible to import the given file

**Steps to recreate the error**:
I created a repository that exemplifies the error, just clone and run the project and you can check the error [(repository)](https://github.com/hmartiins/ErroDeImportacaoNullstack). But to reproduce the error just:

1. Create a Nullstack Component
2. Import this component to some file, but with wrong import path

**Possible suggestion for improvement:**
The error printed on the terminal could be clearer, it could be something like:

```
💥️ Module not found: Can't solve './Exemplo' in ./src/Application.njs
```

**Comment by Mortaro** [comment]
Thanks for the report <3! I have transfered the issue to the Nullstack main repo

**Comment by Mortaro** [comment]
fixed in 0.11.2!
